templates/bake/element/Controller/add.twig

```diff
@@ -22,6 +22,9 @@
     public function add()
     {
         ${{ singularName }} = $this->{{ currentModelName }}->newEmptyEntity();
+{% if Bake.hasPlugin('Authorization') %}
```

Comment:
I am curious about the use case for authorizing using an empty entity. Personally I have only authorized the "add" action.

Reply:
Applications could have roles or rules on who can create what. I agree that most of the time add() will have a policy of `return true`.